schema/deploy/types/reporting_requirement_status.sql

```diff
@@ -0,0 +1,14 @@
+-- Deploy cif:types/reporting_requirement_status to pg
+
+begin;
+
+create type cif.reporting_requirement_status as enum (
```

Comment:
This should be a table, not an enum. An enum would not be editable by users, and we want them to be able to configure this list